static/styles/portico/portico.css

```diff
@@ -572,7 +572,7 @@ input.text-error {
     font-size: 16px;
 }
 
-.table.table-striped {
+table {
```

Comment:
So, this will apply to all tables in the portico, which is not our goal. Part of the goal of removing bootstrap is remove all unscoped CSS like the block you're adding here. We should apply this CSS only to tables inside a portico markdown block.

Reply:
Okay, I removed this CSS and moved one usable line to `markdown.css`.